diff --git a/src/ClustalW/src/general/clustalw.h b/src/ClustalW/src/general/clustalw.h
index 9b0111b..c5255a8 100644
--- a/src/ClustalW/src/general/clustalw.h
+++ b/src/ClustalW/src/general/clustalw.h
@@ -229,7 +229,10 @@ const string VALID_COMMAND_SEP = "-/";
         for profile scores */
 #endif
 
-       
+#ifndef INT_SCALE_FACTOR
+#define INT_SCALE_FACTOR 1000  /* copy the value used by the Unix/Linux branch */
+#endif
+
 const int NUMRES = 32; /* max size of comparison matrix */
 const int INPUT = 0;
 const int ALIGNED = 1;
